cpmonitor/models.py

```diff
@@ -1,51 +1,244 @@
-        "Zustand", choices=States.choices, default=States.UNKNOWN
+    responsible_organ = models.TextField(
+        "Verantwortliches Organ",
+        blank=True,
+        help_text="""
+            <p>Genauer Name des verantwortlichen Gremiums oder der verantwortlichen Behörde.</p>
+            <p>Möglichst mit Ansprechperson, Kontaktdaten und was sonst notwendig ist, um Informationen einzuholen.</p>
+            <p>Diese Informationen könnten später in separate Felder aufgeteilt werden.</p>
+        """,
     )
 
-    justification = MartorField("Begründung Zustand", blank=True, default="")
+    # 2. Bewertung der KAP Inhalte
+
+    plan_assessment = models.TextField(
+        "Bewertung der Planung",
+        blank=True,
+        help_text="""
+            <p>Würde(n) die im Klimaaktionsplan beschriebenen Maßnahme(n) ausreichen, um das gesteckte Ziel zu erreichen?</p>
+            <p>Sollte das gesteckte Ziel nicht ausreichen, kann das auch hier beschrieben werden.</p>
+        """,
+    )
+
+    # 3. Umsetzungsstand
+
+    class ExecutionAssessment(models.IntegerChoices):
+        UNKNOWN = 0, "unbekannt (grau)"
+        AHEAD = 1, "vor dem Plan (grün)"
+        AS_PLANNED = 2, "im Plan (grün)"
+        DELAYED = 3, "zurückgestellt / verzögert (orange)"
+        INSUFFICIENT = 4, "nicht ausreichend / in Teilen gescheitert (rot)"
+        FAILED = 5, "gescheitert (rot)"
+        MISSING = 6, "fehlt im Plan (rot)"
+
+    execution_assessment = models.IntegerField(
+        "Bewertung des Umsetzungsstandes",
+        choices=ExecutionAssessment.choices,
+        default=ExecutionAssessment.UNKNOWN,
+        help_text="""
+            <p>Bei Maßnahmen: Wird/wurde die Maßnahme wie geplant umgesetzt?</p>
+            <p>Bei Sektoren / Maßnahmengruppen:</p>
+            <p>Wenn hier "unbekannt" ausgewählt wird, werden die Umsetzungsstände der Maßnahmen in diesem Sektor / dieser Gruppe zusammengefasst.</p>
+            <p>Bei anderen Auswahlen wird diese Zusammenfassung überschrieben. Das sollte nur passieren, wenn sie unpassend oder irreführend ist.</p>
+        """,
+    )
+
+    execution_justification = models.TextField(
+        "Begründung der Bewertung",
+        blank=True,
+        help_text="Die Auswahl bei der Bewertung kann hier ausführlich begründet werden.",
+    )
+
+    class ExecutionProgress(models.IntegerChoices):
+        UNKNOWN = 0, "unbekannt"
+        NOT_PLANNED = 1, "ungeplant"
+        PLANNED = 2, "in Zukunft geplant"
+        IN_PROGRESS = 3, "läuft"
+        FINISHED = 4, "abgeschlossen / gescheitert"
+
+    execution_progress = models.IntegerField(
+        "Fortschritt",
+        choices=ExecutionProgress.choices,
+        default=ExecutionProgress.PLANNED,
+        help_text="""
+            <p>Eine grobe Einteilung, wo sich diese Maßnahme zeitlich in der Planung befindet.</p>
+            <p>Dies kann z.B. genutzt werden, um Maßnahmen zu filtern, deren Umsetzungsstand noch nicht bewertet werden kann.</p>
```

Comment:
Mir wäre als Benutzer nicht klar wofür ich das hier benutzen soll, und wofür die Bewertung des Umsetzungsstandes. Beides beschreibt irgendeine Form von zeitlichem Fortschritt, aber ich finde es nicht klar verständlich wie beides ineinandergreift oder wie es zusammenpasst.

Reply:
Das ist Teil von den Themen, die wir in zwei Wochen noch ausführlich besprechen wollen.

Reply:
Okay, hab inzwischen die Diskussion auch nachgeholt.
Dann fassen wir das da einfach noch mal an.